### PR TITLE
don't use jq to parse kube version

### PR DESCRIPTION
**Pull request by areed**

### CODE REVIEW

File: install_scripts/templates/kubernetes-init.sh [hunk 1]
```diff
@@ -86,9 +86,9 @@ EOF
 getKubeServerVersion() {
     logStep "check k8s server version"
     # poll until we can get the current server version
-    _current="$(kubectl version --output json | docker run -i --rm realguess/jq jq -r .serverVersion.gitVersion)"
-    until [ "$_current" != "null" ]; do
-      _current="$(kubectl version --output json | docker run -i --rm realguess/jq jq -r .serverVersion.gitVersion)"
+    _current="$(kubectl version -o yaml | grep gitVersion | cut -d" " -f4)"
+    until [ -n "$_current" ]; do
+      _current="$(kubectl version -o yaml | grep gitVersion | cut -d" " -f4)"
```
Comment (dexhorthy) [inline install_scripts/templates/kubernetes-init.sh]: will this ever get the client version and not the server version?

Comment (dexhorthy) [reply install_scripts/templates/kubernetes-init.sh]:
```kubectl version -o yaml                             
clientVersion:
  buildDate: 2018-03-12T16:29:47Z
  compiler: gc
  gitCommit: bee2d1505c4fe820744d26d41ecd3fdd4a3d6546
  gitTreeState: clean
  gitVersion: v1.9.4
  goVersion: go1.9.3
  major: "1"
  minor: "9"
  platform: linux/amd64
serverVersion:
  buildDate: 2018-02-16T18:26:58Z
  compiler: gc
  gitCommit: 6e5b33a290a99c067003632e0fd6be0ead48b233
  gitTreeState: clean
  gitVersion: v1.8.8-gke.0
  goVersion: go1.8.3b4
  major: "1"
  minor: 8+
  platform: linux/amd64
```

Comment (areed) [reply install_scripts/templates/kubernetes-init.sh]: good catch

Comment (dexhorthy) [reply install_scripts/templates/kubernetes-init.sh]: i am sad that this does not support go template or jsonpath

**Pull request merged**